nilearn/datasets/tests/test_atlas.py

```diff
@@ -321,32 +321,40 @@ def test_fetch_coords_seitzman_2018(request_mocker):
     assert np.any(bunch.networks != np.sort(bunch.networks))
 
 
-def _destrieux_data():
+def _destrieux_data(n_labels=10):
     data = {"destrieux2009.rst": "readme"}
+    atlas = np.random.randint(0, 10, (10, 10, 10))
+    atlas_img = nibabel.Nifti1Image(atlas, np.eye(4))
+    # Create 11 labels. The fetcher should be able to remove
+    # the extra label ('10') not present in the atlas image
+    labels = "\n".join([f"{idx},label {idx}" for idx in range(n_labels)])
```

Comment:
Sorry, where is the 11th label created ?

Reply:
Yes, I forgot to remove this comment. Thanks for noticing..
The number of labels in the CSV file is now passed as an argument and used for test parametrization.
I removed the comment and added more informative docstrings for these functions in 16ca0d0.